packages/ra-ui-materialui/src/defaultTheme.ts

```diff
@@ -7,11 +27,6 @@ export default {
             contrastText: '#fff',
         },
     },
-    typography: {
```

Comment:
Why did you remove this one?

Reply:
I didn't mean to push that with this PR but I checked everywhere there is no use for variant `title`